langserver/tracer.py

```diff
@@ -0,0 +1,27 @@
+import opentracing
+import lightstep
+
+
+class Tracer:
+
+    # just use class vars; no need for multiple independent tracer instances
```

Comment:
The usual pattern for tracing is that in our entrypoint we set the global tracer to lightstep, then inside of the project we only refer to opentracing. I would think we should do the same in this repo.